src/ui/templates/index.ts

```diff
@@ -0,0 +1 @@
+export { ContentCentered } from './content-centered';
```

Comment:
Сдавай введем практику, добавлять шаблонам суффикс Template в имя компонента. При этом файл именуем как сейчас

```suggestion
export { ContentCenteredTemplate } from './content-centered';
```